java/ql/lib/ext/org.apache.commons.lang.model.yml

```diff
@@ -0,0 +1,7 @@
+extensions:
+  - addsTo:
+      pack: codeql/java-all
+      extensible: summaryModel
+    data:
+      - ["org.apache.commons.lang", "StringEscapeUtils", true, "escapeHtml", "(String)", "", "Argument[0]", "ReturnValue", "taint", "manual"]
+      - ["org.apache.commons.lang", "StringEscapeUtils", true, "escapeHtml", "(Writer,String)", "", "Argument[1]", "Argument[0]", "taint", "manual"]
```

Comment:
These are good summaries. Nonetheless, we should:
* Remove their neutrals from the generated models: https://github.com/github/codeql/blob/aa59741c38d40ca7f76216d9b9ea29d4c8461fc7/java/ql/lib/ext/generated/org.apache.commons.lang.model.yml#L1607-L1608
* Make sure that we add them as sanitizers to the XSS queries (if they are actually good sanitizers in that regard) so that we don't introduce FPs with these new edges.

Reply:
> Make sure that we add them as sanitizers to the XSS queries (if they are actually good sanitizers in that regard)

Well, as we suspected, these are not enough to sanitize XSS if the context is different than HTML (e.g. an injection directly in JavaScript). So scratch adding them as XSS sanitizers.